backend/src/controllers/reports/Tab_50_rpt_PF_NetRecoverySummaryByQuarter.js

```diff
@@ -0,0 +1,53 @@
+const useCommonComponents = require("../useCommonComponents/index");
+const useController = require("../useController/index");
+const model = require("@models/reports/projectRollup");
+const utils = require("./helpers");
+const what = { single: "report", plural: "reports" };
+const controller = useController(model, what);
+
+// Template and data reading
+const cdogs = useCommonComponents("cdogs");
+const { getReport, getDocumentApiBody, pdfConfig, groupByProperty } = utils;
+controller.getReport = getReport;
```

Comment:
i thought you weren't going to do this to getReport, is it because you separated all your routes as well?